notebook/db_util.py

```diff
@@ -1,20 +1,29 @@
 # Db util class which take care of executing queries.
 
 import os
+import requests
+from requests.auth import HTTPBasicAuth
+from urllib.parse import urljoin
 
 import django
 from django.conf import settings
 
 
-from deming import deming_setup
-
 # PG values.
 PG_DB_USER = os.environ.get('PG_DB_USER', '')
 PG_DB_NAME = os.environ.get('PG_DB_NAME', '')
 PG_DB_PASS = os.environ.get('PG_DB_PASS', '')
 PG_DB_HOST = os.environ.get('PG_DB_HOST', '')
 PG_DB_PORT = os.environ.get('PG_DB_PORT', '')
```

Comment:
Are these required?

Reply:
Data base setup and all is also not required I think you can remove that.

Reply:
Yes, removed